contrib/docker/Makefile

```diff
@@ -25,7 +25,7 @@
 # Override with make PARALLEL="-j <num_parallel_processes>" where
 #    <num_parallel_processes> = the number of make processes to run in parallel
 # Turn off with make PARALLEL=
-PARALLEL=-j 22 
+PARALLEL=22 
```

Comment:
One negative I see with changing PARALLEL to be only the number of threads is that you can never run _make_ without the -j option.  Also, you can not easily run _make_ with the default number of threads for -j, since the build-and-test script will set PARALLEL=22 if the PARALLEL option is empty (but set).

Reply:
Agreed.  I restored the original setting methodology.